ecommerce-ui/src/components/Location/index.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+
+class Location extends Component {
+
+  render() {
+    const{
+      title,
+      houseType,
+      image,
+      location,
+      payment,
+      host,
+      rating 
+    } = this.props.airbandbs;
+    
+    console.log(this.props.airbandbs)
+    
+    return(
+
+      <div className="location">
+        <img src={image} alt={title} />
+        <h2>{title} </h2>
+        <p>{houseType} • {location.city}</p>
+        <p>Host: {host.name} <br /> </p>
+        <button className="btn">Add to Cart</button>
```

Comment:
<button type="button" onClick={(e) => this.props.handleAdd(this.props.item.title, this.props.item.payment.cost)}>Add to cart</button>